cl/settings/third_party/celery.py

```diff
@@ -6,6 +6,21 @@
 DEVELOPMENT = env.bool("DEVELOPMENT", default=True)
 CELERY_ETL_TASK_QUEUE = env("CELERY_ETL_TASK_QUEUE", default="celery")
 CELERY_IQUERY_QUEUE = env("CELERY_IQUERY_QUEUE", default="celery")
+CELERY_QUEUES = (
+    "batch0",
+    "batch1",
+    "batch2",
+    "batch3celery",
```

Comment:
According to https://github.com/freelawproject/kubernetes/pull/34/ this should be:
```suggestion
    "batch3",
```

correct?